Sources/Exporters/OpenTelemetryProtocolCommon/metric/MetricsAdapter.swift

```diff
@@ -247,6 +249,7 @@ public enum MetricsAdapter {
         protoExemplar.spanID = TraceProtoUtils.toProtoSpanId(spanId: spanContext.spanId)
         protoExemplar.traceID = TraceProtoUtils.toProtoTraceId(traceId: spanContext.traceId)
       }
+      protoPoint.exemplars.append(protoExemplar)
```

Comment:
this plus the above lines can probably be removed since exemplars don't exist for summary data points

Reply:
Thanks for the feedback! I removed this and the `pointData.exemplars.forEach` above assuming that's what you were suggesting.